readme.md

```diff
@@ -41,7 +41,7 @@ See [`example.js`](example.js) if you want to quickly try it out. *(The example
 
 ## API
 
-### aperture.screens() -> `Promise<Object[]>`
+#### aperture.screens() -> `Promise<Object[]>`
```

Comment:
Why make this smaller? Now it also skips a semantic step from `## API`.

Reply:
There already was a bit of an issue with semantics, so of the two evils I preferred this since made the document easier to read.